packages/client/src/members/verify.ts

```diff
@@ -0,0 +1,31 @@
+import { MemberId } from "@raha/api-shared/models/identifiers";
```

Comment:
Make sure this refers to `dist/` version, i'd rebase or merge in master before merging this